armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -51,6 +51,12 @@ class GlobalFluxInterface(interfaces.Interface):
     function = "globalFlux"
     _ENERGY_BALANCE_REL_TOL = 1e-5
 
+    # for tight coupling, overwrite base class
+    # TODO: expose tightCouplingConvergeOn to user and allow flexibility via case settings
```

Comment:
Good rule of thumb: let's not create PRs with `TODO`s in them. Usually, it's a good sign that the PR isn't quite done yet.

Reply:
Yep! It's definitely not done yet 😄 . Just wanted to get reviews going on what I have.